README.md

```diff
@@ -15,42 +15,97 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-[![Apache Accumulo][logo]][accumulo]
---
-[![Build Status][ti]][tl] [![Maven Central][mi]][ml] [![Javadoc][ji]][jl] [![Apache License][li]][ll]
-
-[Apache Accumulo][accumulo] is a sorted, distributed key/value store that provides robust,
-scalable data storage and retrieval. With Apache Accumulo, users can store and manage large
+<!-- LOGO -->
+<p align="center">
+    <a href="https://vuejs.org" target="_blank" rel="noopener noreferrer">
+        <img src="contrib/accumulo-logo.png" alt="Vue logo">
+    </a>
+</p>
+
+<!-- BADGES -->
+<p align="center">
+    <!-- Build Status -->
+    <a href="https://github.com/apache/accumulo/actions">
+        <img src="https://github.com/apache/accumulo/workflows/QA/badge.svg" alt="Build Status">
+    </a>
+    <!-- Maven Central -->
+    <a href="https://maven-badges.herokuapp.com/maven-central/org.apache.accumulo/accumulo-core">
+        <img src="https://maven-badges.herokuapp.com/maven-central/org.apache.accumulo/accumulo-core/badge.svg"
+            alt="Maven Central">
+    </a>
+    <!-- JavaDocs -->
+    <a href="https://www.javadoc.io/doc/org.apache.accumulo/accumulo-core">
+        <img src="https://www.javadoc.io/badge/org.apache.accumulo/accumulo-core.svg" alt="JavaDocs">
+    </a>
+    <!-- Apache License -->
+    <a href="https://www.apache.org/licenses/LICENSE-2.0">
+        <img src="https://img.shields.io/badge/license-ASL-blue.svg" alt="Apache License">
+    </a>
+</p>
+
+# About
```

Comment:
I'm opposed to expanding the README to duplicate information on the website. We previously took efforts to substantially minimize the README to make it easier to read, and keep more comprehensive information up-to-date on the website instead. The README really should be minimal, so we don't have to maintain the same information in multiple places. A link to the website should be fine.

Keep in mind that the README is also included in the release tarball, so links to files/pages not in the tarball won't work. That includes everything in `.github/`.

Also, links to specific pages on the website should be minimized. Since this file is included in the release tarball at the time of release, it should endeavor to show information that will not be invalidated by website redesigns or reorganizations.

Reply:
> I'm opposed to expanding the README to duplicate information on the website. We previously took efforts to substantially minimize the README to make it easier to read, and keep more comprehensive information up-to-date on the website instead. The README really should be minimal, so we don't have to maintain the same information in multiple places. A link to the website should be fine.

I see well that the sections are maintained ... Opinions ... :)

> Keep in mind that the README is also included in the release tarball, so links to files/pages not in the tarball won't work. That includes everything in .github/.
> Also, links to specific pages on the website should be minimized. Since this file is included in the release tarball at the time of release, it should endeavor to show information that will not be invalidated by website redesigns or reorganizations.

It seems good to me I will check the indexing of url's and that they comply with this commitment